android/app/src/main/java/org/fossasia/openevent/activities/BaseActivity.java

```diff
@@ -52,6 +52,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
                 onBackPressed();
+                break;
+            default:
```

Comment:
Please add a comment in default block explaining that nothing is to be done here